RecoMuon/Configuration/python/DisplacedMuonSeededStep_cff.py

```diff
@@ -60,7 +60,6 @@
 muonSeededTracksOutInDisplacedClassifier = RecoTracker.IterativeTracking.MuonSeededStep_cff.muonSeededTracksOutInClassifier.clone()
 muonSeededTracksOutInDisplacedClassifier.src='muonSeededTracksOutInDisplaced'
 
-
```

Comment:
There should be less packages to be checked out if you add the missing line back (for the final PR you may want to squash commits together).